lib/Partitioner/Partitioner.cpp

```diff
@@ -16,6 +16,23 @@
 
 #include "glow/Partitioner/Partitioner.h"
 
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/raw_ostream.h"
+
+/// -log-partition - Command line option to dump Partitioner logs.
+static llvm::cl::OptionCategory PartitionerCat("Glow Partitioner Options");
+static llvm::cl::opt<bool>
+    logPartition("log-partition",
+                 llvm::cl::desc("Enable logging partition info"),
+                 llvm::cl::init(false), llvm::cl::cat(PartitionerCat));
+
+/// -dump-partition - Command line option to dump the graph of each partitions
+/// by calling F->dumpDAG().
+static llvm::cl::opt<bool>
```

Comment:
I'm fine to keep this knob, but logging info for `"log-partition"` would be extremely useful for each run.

Reply:
BTW, there is also always a possibility to use something like:
```
#define DEBUG_TYPE "partitioner"
...
DEBUG_GLOW(LOG(INFO) << "...");
```

This does not require any new options. If you want to enable this debug prints, you just use `-debug-glow-only=partitioner`.

Reply:
Thanks a lot! Keep it in my notebook :) 
The log is enabled by default now.